components/src/Button/Button.js

```diff
@@ -159,9 +176,8 @@ const StyledButton = styled.button`
 
   &:active {
     border-style: solid;
-    box-shadow: none;
+    //box-shadow: none;
```

Comment:
delete

Reply:
I removed ripple effect and removed comment here since this line of code is needed in absence of ripple effect.